code-challenges.test.js

```diff
@@ -1,5 +1,7 @@
 // ASSESSMENT 3: Coding Practical Questions with Jest
 
+const { expect } = require("@jest/globals")
```

Comment:
This is helper code from your text editor. it is called yargs. If you get errors that state you don't have a test in your test suite, then definitely look to see if you save and look at the top of your file for code like on Lines 3, comment it out and you should be able to test out the info.